armi/reactor/components/complexShapes.py

```diff
@@ -14,6 +14,13 @@
 
 """
 Components represented by complex shapes, and typically less widely used.
+
+.. impl:: ARMI shall support a reasonable set of basic shapes.
```

Comment:
Same name as previous implementation. Maybe better to title this one "Add support for Holed hexes, rectangles, squares, and helices" and remove the follow-up comment